portfolio/src/main/webapp/profiles.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Profile Links </title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <h1>Ethan's Profiles</h1>
+    <p>Here is a list of links to my online profiles. </p>
```

Comment:
Super nitpick: indent the `<p>` and `<a>` tags another 2 spaces to keep them inline with the `<h1>`

Reply:
Got it.